### PR TITLE
Added Mimestream latest.

### PR DESCRIPTION
**Pull request by aren55555**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/mimestream.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask "mimestream" do
+  version "0.6.17"
+  sha256 "f26f4d50e8b7a655289e074da253fbf15c47093386ab9cbefe6ab18a25dfc193"
+
+  # storage.googleapis.com was verified as official when first introduced to the cask
+  url "https://storage.googleapis.com/mimestream-releases/Mimestream_#{version}.dmg"
```
Comment (miccal) [inline Casks/mimestream.rb:6]: Please add the following `appcast`:
```
appcast "https://mimestream.com/releases"
```

File: Casks/mimestream.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  name "Mimestream"
+  desc "Native email client for Gmail"
+  homepage "https://mimestream.com/"
+
```
Comment (miccal) [inline Casks/mimestream.rb:10]: Please add:
```
depends_on macos: ">= :catalina"
```

File: Casks/mimestream.rb (hunk 1 above)
Comment (aren55555) [reply Casks/mimestream.rb:6]: Done.

File: Casks/mimestream.rb (hunk 2 above)
Comment (aren55555) [reply Casks/mimestream.rb:11]: Done.

**Comment by vitorgalvao** [comment]
Thank you, but this is a [walled build](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) so [we will not accept it](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#unofficial-vendorless-and-walled-builds).

**Pull request closed without merging**